### PR TITLE
kube-proxy Nftables GCE jobs

### PR DESCRIPTION
**Pull request by aojea**
Add jobs using GCE for the new kube-proxy nftables mode

### CODE REVIEW

**Review by danwinship — COMMENTED** [review]
yeah lgtm assuming it was mostly copied from another working job

File: config/jobs/kubernetes/sig-network/sig-network-misc.yaml [hunk 1]
```diff
@@ -207,6 +207,67 @@ presubmits:
...
       description: Uses kubetest to run e2e Conformance, SIG-Network tests against a cluster using IPVS (ubuntu based and uses containerd) created with cluster/kube-up.sh
 
+  - name: pull-kubernetes-e2e-gci-gce-nftables
+    cluster: k8s-infra-prow-build
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    always_run: false
+    run_if_changed: '^(test/e2e/network/|pkg/apis/networking|pkg/.*/nftables/)'
```
[the remark this answers is not in the document]
Comment (aojea) [reply config/jobs/kubernetes/sig-network/sig-network-misc.yaml:217]: yeah, I copied it from there

[the remark this answers is not in the document]
Comment (aojea) [reply config/jobs/kubernetes/sig-network/sig-network-misc.yaml]: updated to remove the asterisk

**Pull request merged**